netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1275,13 +1239,70 @@ _net_ebpf_ext_process_redirect_verdict(
     NET_EBPF_EXT_RETURN_NTSTATUS(status);
 }
 
-/*
- * Default action is BLOCK. If this callout is being invoked, it means at least one
- * eBPF program is attached. Hence no connection should be allowed unless allowed by
- * the eBPF program.
+/**
+ * @brief This function determines if the sock_addr eBPF program should be invoked as part of processing the classify
+ * callback at CONNECT_REDIRECT layer.
+ *
+ * @param[in] filter_context Pointer to net_ebpf_extension_sock_addr_wfp_filter_context_t associated with WFP filter.
+ * @param[in] sock_addr_ctx Pointer to bpf_sock_addr_t struct to be passed to eBPF program.
+ * @param[in] v4_mapped Boolean indicating if the IP address in sock_addr is v4 mapped v6 address or not.
  *
- * If the eBPF program verdict is BLOCK, no context is created and AUTH classify will
- * block the connection by default.
+ * @returns True if eBPF program should be invoked, False otherwise.
+ */
+
+bool
+_net_ebpf_extension_sock_addr_should_invoke_ebpf_program(
+    _In_ const net_ebpf_extension_sock_addr_wfp_filter_context_t* filter_context,
+    _In_ const bpf_sock_addr_t* sock_addr_ctx,
+    bool v4_mapped)
+{
+    bool process_classify = TRUE;
+
+    //  If the callout is invoked for v4, then it is safe to invoke the eBPF program.
+    if (sock_addr_ctx->family == AF_INET)
```

Comment:
missing braces?

Reply:
This is compliant to the styling guidelines. I rely on the clang-formatting tool for styling violations :-)

Reply:
Not sure I follow?  [This](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#formatting-notes)  section (that I'd mentioned in an earlier comment) from our 'DevelopmentGuide.md' does call this out.  Genuine question: Is there some other document that you're referring to?

As for clang-format, the version (v11) that we're currently using does not have this support, hence the explicit callout in the Dev guide document.

Reply:
Thanks for pointing out, I have fixed all the instances. Do you think the clang_formatting tool (`format-code.ps1`) can be updated to prevent commit on such changes?